InfoTeamProject21/InfoTeamProject21/Disease.vb

```diff
@@ -1,3 +1,43 @@
-﻿Public Class Disease
+﻿Option Strict On
+Option Explicit On
+Option Infer Off
+Public Class Disease
+    '<<Attributes>>
+    Private _NumberInfected As Integer
+    Private _DeathRate As Integer
+    Private _Name As String
 
+    '<<Property>>
+    Public Property NumberInfected() As Integer
```

Comment:
@Ruan94bez This is not an array, the brackets should be removed.

Reply:
Please change _DeathRate to a double @Ruan94bez, since it is a rate